src/bt_factory.cpp

```diff
@@ -85,7 +87,7 @@ void BehaviorTreeFactory::registerSimpleCondition(const std::string& ID,
         return tick_bool_functor(parent_node) ? NodeStatus::SUCCESS : NodeStatus::FAILURE;
     };
 
-    registerSimpleAction(ID, tick_functor, ports);
+    registerSimpleCondition(ID, tick_functor, ports);
```

Comment:
Why is it renamed?

Reply:
I worry if this change will leave compile error if the old-name function is used

Reply:
This was a bug, it was classifying all condition nodes as action nodes. We needed this change to tell the difference between condition and action nodes.

Reply:
I think I misunderstood your comment, this function was calling the wrong function, I didn't rename registerSimpleAction to registerSimpleCondition they both existed prior

Reply:
I see the context. Maybe create a new commit just for this